Makefile

```diff
@@ -1,35 +1,12 @@
-shell: ## run a shell on the cookie-cutter container
-	docker exec -it edx.gradebook /bin/bash
+shell: ## run a shell
```

Comment:
Will I be able to do `make gradebook-shell` from inside devstack now?

Reply:
Yeah you can get to shell via devstack as well I just left it as a shortcut.  Ultimately, it's unnecessary.  However there's no equivalent of the npm install shortcut in devstack so we probably want to keep that.